### ISSUE TITLE
How to use this offline ? thanks

### ISSUE THREAD

**Issue by sherry0429** [CompVis/stable-diffusion#386]
I have success run it on my computer,

but I find every time when I run it, 

will log this:

Fetching 16 files: 100%|██████████| 16/16 [00:31<00:00,  1.97s/it]

even if I have download model, 

and if I close internet connection, script will raise error.

Is this possible to run this on offline local computer ? thanks

**Comment by samheather** [comment]
Yeah - there's quite a few clients out there to run locally. One I helped with is a 1-click install for M1/M2 Mac's: https://avoloapp.com/ - the owner is giving out free licences on Reddit at the moment
